`--from-playground` & PlaygroundLayout

jycouet opened the pull request:
Here is the result: 
<img width="1995" height="988" alt="image" src="https://github.com/user-attachments/assets/846a6847-42fe-44b5-951f-0ecf31aa9faf" />

## Summary
- [x] a new file `/src/lib/PlaygroundLayout.svelte` to bring the Playground look and feel.
  - [x] title: name of the playground
  - [x] link: to the online playground
  - [x] theme switch available
- [x] change files location to `src/lib/playground`

manuel3108 commented:
Probably because we did not need to parse the file? I think either the parsing or the printing is doing strang stuff, since we are currently not using the svelte compiler

jycouet commented:
> Probably because we did not need to parse the file? I think either the parsing or the printing is doing strang stuff, since we are currently not using the svelte compiler

Yeah, so I updated the code to still use it _(to have `lang="ts"` or not)_ + reduce the template as much as possible + doing a replace. Not that nice, but working workaround.
I added a few tests to make sure the output is ok.

manuel3108 reviewed (approved):
Works great now!
